src/main/java/ca/nines/ise/writer/XMLWriter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 Michael Joyce <ubermichael@gmail.com>
+ * Copyright (C) 2014 Malcolm Moran <malcolm.moran@outlook.com>
```

Comment:
I think Malcolm basically threw-out the existing code and started from scratch...